packages/cli/src/info/env_info_types.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+export type EnvInfoBinary = {
```

Comment:
```suggestion
// Types in this file are for return type from 'envinfo' package. We don't own schema.
/* eslint-disable @typescript-eslint/naming-convention */
export type EnvInfoBinary = {
```

Please add comment explaining why.